crates/jxl-oxide/fuzz/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "jxl-oxide-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+honggfuzz = "0.5.55"
+jxl-oxide = {path = ".."}
+
+# Prevent this from interfering with workspaces
+[workspace]
+members = ["."]
```

Comment:
How about having the fuzzer as a workspace member? You can move this crate to `crates/jxl-oxide-fuzz` and add it to toplevel `Cargo.toml`.

Reply:
Sounds like a good idea. Will do that!